src/components/ListItem/ListItem.tsx

```diff
@@ -108,10 +111,13 @@ const ListItem: FC<ListItemProps> & { sizes?: typeof SIZES } = forwardRef(
 
     return (
       <div
+        data-testid={dataTestId || getTestId(ComponentDefaultTestId.LIST_ITEM, id)}
         ref={mergedRef}
-        className={cx("list-item", className, BEMHelper(size.toString()), {
+        className={cx(styles.listItem, "list-item", className, BEMHelper(size.toString()), {
```

Comment:
In my edition that all looked like this 
![image](https://user-images.githubusercontent.com/104433616/225286874-d229bfe5-011e-4ef4-985e-a62cc824fc74.png)